src/app/containers/Article/index.jsx

```diff
@@ -29,6 +30,7 @@ const ArticleContainer = ({ loading, error, data }) => {
         <GlobalStyle />
         <ServiceContextProvider service={service}>
           <PlatformContextProvider platform={isAmp ? 'amp' : 'canonical'}>
+            <ConsentBanner />
```

Comment:
Does this not need to be dependant on isAmp?

Reply:
Ahh its in the container. Hmmm this could get confusing

Reply:
The ConsentBanner container should be able to render both AMP and Canonical versions.